agent/runner/actions/mysql_explain_action_test.go

```diff
@@ -205,12 +209,9 @@ func TestMySQLExplain(t *testing.T) {
 			Query:        `INSERT INTO city (Name)...`,
 			OutputFormat: agentpb.MysqlExplainOutputFormat_MYSQL_EXPLAIN_OUTPUT_FORMAT_DEFAULT,
 		}
-		a := NewMySQLExplainAction("", time.Second, params)
-		ctx, cancel := context.WithTimeout(context.Background(), a.Timeout())
-		defer cancel()
-
-		_, err := a.Run(ctx)
-		require.Error(t, err, "EXPLAIN failed because the query was too long and trimmed. Set max-query-length to a larger value.")
+		a, err := NewMySQLExplainAction("", time.Second, params)
+		assert.Error(t, err, "EXPLAIN failed because the query was too long and trimmed. Set max-query-length to a larger value.")
```

Comment:
I'm not sure if `assert.Error` checks if error is equal to the last argument. it's better ot use `assert.EqualError`

Reply:
Yep, this is an error.